concordia/static/scss/action-app.scss

```diff
@@ -318,9 +315,7 @@ main {
 #review-transcription-text {
     border: 1px solid $gray-800;
     padding: 10px;
-    flex-grow: 1;
-    flex-shrink: 1;
-    max-height: 100%;
+    flex: 1 1 0;
```

Comment:
I generally prefer not using the shorthand forms but this is otherwise fine by me